libraries/AP_Relay/AP_Relay_Params.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <AP_Param/AP_Param.h>
+#include "AP_Relay_config.h"
+
+class AP_Relay_Params {
+public:
+    static const struct AP_Param::GroupInfo var_info[];
+
+    AP_Relay_Params(void);
+
+    /* Do not allow copies */
+    CLASS_NO_COPY(AP_Relay_Params);
+
+    enum class DEFAULT_STATE : uint8_t {
```

Comment:
Usually it would be `DefaultState` here - so class-name-ish for the enum-class name.  The enum entries *I* like to be upper-case....

Reply:
Fixed.